server/app.js

```diff
@@ -9,6 +9,7 @@ const router = require('./routes/router');
 const httpRequestsLogger = require('./middleware/httpRequests.middleware');
 const errors = require('./middleware/errors.middleware');
 
+app.use(helmet);
```

Comment:
I'm getting errors on my console when I launch the app, and the server does not populate the map. I believe this should be app.use(helmet())